libs/api/domains/application/src/lib/application.model.ts

```diff
@@ -131,6 +131,30 @@ export class Application {
   status!: ApplicationResponseDtoStatusEnum
 }
 
+@ObjectType()
+export class ApplicationStatistics {
+  @Field(() => String)
+  typeid!: string
```

Comment:
Should we use the friendly display name here ?

Reply:
do you mean typeId instead of typeid?

Reply:
No just meant to use the display name from the Template for displaying purposes. Like the template typeId `CriminalRecord`  would have the display name `Sakavottorð` but for this first iteration I think we should leave this as is.

Reply:
Ah, gotcha. Makes sense.